rust/rsc/src/rsc/add_job.rs

```diff
@@ -44,49 +47,67 @@ pub async fn add_job(
                 let job = insert_job.save(txn).await?;
                 let job_id = job.id.unwrap();
 
-                // TODO: Convert these loops to singular add_many calls
+                let mut visible_files = Vec::new();
                 for vis_file in vis {
```

Comment:
let visible_files = vis.iter().map(|visible_file| {
    visible_file::ActiveModel {
                        id: ActiveValue::NotSet,
                        path: ActiveValue::Set(vis_file.path),
                        hash: ActiveValue::Set(vis_file.hash.into()),
                        job_id: ActiveValue::Set(job_id)
                        }
}).collect();

Reply:
Also lol, apparently I had a TODO for this already